src/components/forms/news-form/news-form.js

```diff
@@ -147,65 +135,71 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
-              buttonLabel={AUTHOR_PHOTO}
-            />
+        {navButtons}
+        <Grid item xs={12}>
+          <Paper className={styles.inputBlock}>
+            <div className={styles.imageUploadBlock}>
+              <div>
+                <span className={styles.imageUpload}>
+                  {config.labels.news.avatarText}
+                </span>
 
-            {authorAvatar && (
-              <Avatar src={authorAvatar}>
-                <Image />
-              </Avatar>
-            )}
-          </div>
-        </Box>
-        <Box my={3}>
-          <div className={styles.imageUploadAvatar}>
-            <ImageUploadContainer
-              handler={handleImageLoad}
-              buttonLabel={MAIN_PHOTO}
-            />
-            {newsAvatar && (
-              <Avatar src={newsAvatar}>
-                <Image />
-              </Avatar>
-            )}
-          </div>
-        </Box>
-        {preferredLanguages.length > 0
-          ? preferredLanguages.map((lang) => (
-            <LanguagePanel
-              lang={lang}
-              inputOptions={inputOptions}
-              key={lang}
-            />
-          ))
-          : null}
-      </form>
+                <div className={styles.imageUploadAvatar}>
+                  <ImageUploadPreviewContainer
+                    handler={handleImageLoad}
+                    src={uploadAuthorImage}
+                    id={imageUploadNewsInputsId.authorImageInput}
+                  />
+                  {touched.uploadAuthorImage && errors.uploadAuthorImage && (
+                    <div className={styles.inputError}>
+                      {errors.uploadAuthorImage}
+                    </div>
+                  )}
+                </div>
+              </div>
+
+              <div>
+                <span className={styles.imageUpload}>
+                  {config.labels.news.mainImgText}
+                </span>
 
-      <div className={styles.controlsBlock}>
-        <StandardButton
-          id='back-btn'
-          title={GO_BACK_TITLE}
-          variant='outlined'
-          onClickHandler={handleGoBack}
-          data-cy='back-btn'
-        />
-      </div>
+                <div className={styles.imageUploadAvatar}>
+                  <ImageUploadPreviewContainer
+                    handler={handleImageLoad}
+                    src={uploadNewsImage}
+                    id={imageUploadNewsInputsId.newsImageInput}
+                  />
+                  {touched.patternConstructorImage &&
+                    errors.patternConstructorImage && (
+                    <div className={styles.inputError}>
+                      {errors.patternConstructorImage}
+                    </div>
+                  )}
+                </div>
+              </div>
+            </div>
+          </Paper>
+        </Grid>
+        {map(languages, (lang) => (
+          <LanguagePanel lang={lang} inputOptions={inputOptions} key={lang} />
+        ))}
+        {navButtons}
```

Comment:
or this

Reply:
Improved UI, there are a lot of inputs and you need to scroll a lot to go backwards or save the news.
As an option, I can make the position fixed as we have save/delete while editing a product.

Reply:
Made position fixed for save/delete buttons